Attribute Analysis.ipynb

```diff
@@ -971,6 +974,122 @@
     "interact(cooccurrence_counts_mi, topn=widgets.IntSlider(min=1, max=30, step=1, value=10));"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# att_scn Metric: Scenes\n",
+    "<a id=\"att_scn\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "<a id=\"att_scn_setup\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hide_toggle(for_next=True, toggle_text='Show/hide att_scn code')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stats_info = pickle.load(open('results/{}/att_scn.pkl'.format(folder_name), 'rb'))\n",
+    "scenes_per = stats_info['scenes_per']\n",
+    "print(len(scenes_per[0]))\n",
+    "info = pickle.load(open('util_files/places_scene_info.pkl', 'rb'))\n",
+    "idx_to_scene = info['idx_to_scene']\n",
+    "idx_to_scenegroup = info['idx_to_scenegroup']\n",
+    "sceneidx_to_scenegroupidx = info['sceneidx_to_scenegroupidx']\n",
+    "\n",
+    "xaxis = [idx_to_scenegroup[i] for i in range(len(idx_to_scenegroup))]\n",
+    "xaxis = ['\\n'.join(textwrap.wrap(chunk, width=30)) for chunk in xaxis]\n",
+    "barWidth = .6\n",
+    "fontsize=10\n",
+    "\n",
+    "fig = plt.figure(figsize=(10, 5))\n",
+    "r1 = np.arange(len(scenes_per))\n",
+    "\n",
+    "scenes_attribute = [None]*num_attrs\n",
+    "for a in range(num_attrs):\n",
+    "    scenes_attribute[a] = np.array([scenes_per[i][a] for i in range(len(r1))]) / dataset.num_skin_tone_images[a]\n",
+    "    \n",
+    "def show_scenes():\n",
+    "    fig = plt.figure(figsize=(10, 5))\n",
+    "    \n",
+    "    scene_attr_ratios = [0 for i in range(len(scenes_attribute[0]))]\n",
+    "    for scene in range(len(scenes_attribute[0])):\n",
+    "        max_scene = 0\n",
+    "        min_scene = 1\n",
```

Comment:
pretty sure you can't set min_scene to 1 here, it should be like float('inf') or something?